store/index.js

```diff
@@ -165,7 +165,11 @@ export const mutations = {
     state.isNetworkListenerInit = val
   },
   setNetworkStatus(state, val) {
-    state.networkConnected = val.connected
```

Comment:
I'm confused about this change actually. Was `val.connected` false in your testing when `connectionType` was not `none`?